docs/infrastructure/logging.md

```diff
@@ -257,6 +264,116 @@ determines how often logger should send its collected events to sentry.io
 service. If `debug` value is `true`, logging will print to stdout additional
 trace information of the logging process itself.
 
+## Logging server
+
+The centralised logging service is implemented as a logging server started by
+the
+[Project Manager](https://github.com/enso-org/enso/blob/c47dba1d108e0d52e401333d1b6f6f4182436aa7/lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/ProjectManager.scala#L326-L345).
+The implementation of logging server is in
+[org.enso.logging.service.logback.LoggingServer](https://github.com/enso-org/enso/blob/da6a6ae9dde9c0cef02c1d07be5e03660b40d086/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/LoggingServer.java).
+The logging server
+[listens](https://github.com/enso-org/enso/blob/da6a6ae9dde9c0cef02c1d07be5e03660b40d086/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/SocketServer.java#L56)
+to clients that connect via a socket. The client is, for example, language
```

Comment:
How does the `enso` process find out which port (and possibly also a host) to connect to? Is there some environment variable? If so, it deserves to be documented here.

Reply:
CC @hubertp. Are you willing to put your knowledge here? Because I have no idea.